### PR TITLE
Index Page For Testing

### PR DESCRIPTION
**Pull request by spillink**
I have created a placeholder for `index.html` to allow local testing using `python3 -m http.server 8000` to serve `http://localhost:8000/`

### CODE REVIEW

**Comment by spillink** [comment]
I was able to test the site with this change.

File: index.html [hunk 1]
```diff
@@ -1 +1,31 @@
...
+            </a>
+        </li>
+        <li class="clickable-item">
+            <a href="#">
+                <img src="../res/sample_image.jpg" alt="Item 2">
+                <span>Item 2</span>
+            </a>
+        </li>
+        <!-- Add more items as needed -->
+    </ul>
```
Comment (JS0224) [inline index.html:26]: I think we should make this with javascript code instead of hard-coding

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const firstItem = document.querySelector('.clickable-item:first-child a');
+
+if (firstItem) {
+    firstItem.addEventListener('click', function (e) {
+        e.preventDefault(); // Prevent the default link behavior
+        window.location.href = this.getAttribute('href'); // Navigate to start.html
+        test_id = 1;
+        lan = 'kor';
+    });
+}
```
Comment (JS0224) [inline js/index.js:16]: what happens if someone click second item instead of 1st item

Comment (spillink) [reply js/index.js:16]: Nothing. I needed this page for testing. I plan to make it better later on.

File: index.html (hunk 1 above)
Comment (spillink) [reply index.html:26]: We should, but as I have written above, this is a placeholder... and I don't see why these need to be in javascript code.

**Comment by spillink** [comment]
I need this page for testing, so I am going to merge it. If anything needs addressing, make tickets and assign me.

**Pull request merged**